### PR TITLE
Initial files for move to cloudfoundry

### PR DESCRIPTION
**Pull request by AndrewTorrance**
## Checklist
  -  CHANGELOG.md updated?

### CODE REVIEW

**Comment by AndrewTorrance** [comment]
[2nd 1/2 SDX into Cloudfoundry (5)](https://trello.com/c/bB87Q2Se/522-2nd-1-2-sdx-into-cloudfoundry-5)

File: manifest.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+applications:
+- name: sdx-transform-cora
+  buildpack: python_buildpack
+  health-check-type: process
+
+  env:
+    CF_DEPLOYMENT: True
+    SDX_SEQUENCE_URL: http://sdx-sequence.apps.devtest.onsclofo.uk/sequence
+    FTP_PATH: \\\\NP3RVWAPXX370\\SDX_preprod\\
```
Comment (warrenbailey) [inline manifest.yml]: Since it's not preprod, I don't think we need this path. It probably can just be "\"

File: transform/settings.py [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 
-SDX_SEQUENCE_URL = os.getenv("SDX_SEQUENCE_URL", "http://sdx-sequence:5000")
-
-FTP_PATH = os.getenv("FTP_PATH", "\\\\NP3RVWAPXX370\\SDX_preprod\\")
 SDX_FTP_IMAGES_PATH = "EDC_QImages"
 SDX_FTP_DATA_PATH = "EDC_QData"
 SDX_FTP_RECEIPT_PATH = "EDC_QReceipts"
+
+
+if not os.getenv("CF_DEPLOYMENT", False):
```
Comment (warrenbailey) [inline transform/settings.py]: You'll still need these set regardless

File: transform/settings.py [hunk 3]
```diff
@@ -1,12 +1,16 @@
+import json
 import logging
 import os
 
 LOGGING_FORMAT = "%(asctime)s|%(levelname)s: sdx-transform-cora: %(message)s"
 LOGGING_LEVEL = logging.getLevelName(os.getenv('LOGGING_LEVEL', 'DEBUG'))
 
-SDX_SEQUENCE_URL = os.getenv("SDX_SEQUENCE_URL", "http://sdx-sequence:5000")
```
Comment (warrenbailey) [inline transform/settings.py:7]: And I think you need this default value for when it runs in compose

**Review by warrenbailey — CHANGES_REQUESTED** [review]

File: transform/settings.py (hunk 3 above)
Comment (AndrewTorrance) [reply transform/settings.py:7]: Done

**Review by warrenbailey — APPROVED** [review]

**Pull request merged**